contracts/Everlong.sol

```diff
@@ -307,35 +299,99 @@ contract Everlong is IEverlong {
     // │ Rebalancing                                             │
     // ╰─────────────────────────────────────────────────────────╯
 
-    // TODO: Handle case where rebalancing would exceed gas limit
-    // TODO: Handle when Hyperdrive has insufficient liquidity.
+    // NOTE: Errors from hyperdrive are not handled. The keeper must configure
```

Comment:
It would be good to make this NOTE a `@dev` comment. That will ensure that it's included in things like auto-generated docs.